regparser/web/jobs/utils.py

```diff
@@ -24,6 +28,19 @@ def queue_eregs_job(args, timeout=60*30, result_ttl=-1):
                              result_ttl=result_ttl)
 
 
+def send_notification_email(email_address, status_url):
+    backend = django_settings.EMAIL_BACKEND
+    connection = get_connection(backend=backend)
+    send_mail(status_url, "Job finished at %s" % status_url,
+              "notifications@18F.gov", ["tadhg.ohiggins@gsa.gov"],
```

Comment:
Heh, yes, it should be, and I did mean to make the sender a config variable as well.